CodeEditModules/Modules/AppPreferences/src/Sections/AccountsPreferences/Accounts/Login/GitAccountItem.swift

```diff
@@ -14,7 +14,7 @@ struct GitAccountItem: View {
 
     var body: some View {
         HStack {
-            Image(systemName: "xmark.square.fill")
+            Image("vault.fill")
```

Comment:
Please update to use the new `CodeEditSymbols` package. 

Reference:
* #472 
* https://github.com/CodeEditApp/CodeEditSymbols